cli/man/splinter-authid-update.1.md

```diff
@@ -0,0 +1,113 @@
+% SPLINTER-AUTHID-UPDATE(1) Cargill, Incorporated | Splinter Commands
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+NAME
+====
+
+**splinter-authid-update** — Updates an authorized identity on a Splinter node
+
+SYNOPSIS
+========
+**splinter authid update** \[**FLAGS**\] \[**OPTIONS**\]
+
+DESCRIPTION
+===========
+Updates an existing authorized identity used for accessing the Splinter REST
+API. This command allows the user to change the identity's set of roles.
+
+FLAGS
+=====
+`-h`, `--help`
+: Prints help information
+
+`-q`, `--quiet`
+: Decrease verbosity (the opposite of -v). When specified, only errors or
+  warnings will be output.
+
+`-V`, `--version`
+: Prints version information
+
+`-v`
+: Increases verbosity (the opposite of -q). Specify multiple times for more
+  output.
+
+`--rm-all`
+: Remove all of the roles currently associated with the authorized identity.
+
+`-f`, `--force`
+: Ignore errors based on duplicate values or adding and removing the same
+  role.
+
+OPTIONS
+=======
+`-k`, `--key` PRIVATE-KEY-FILE
+: Specifies the private signing key (either a file path or the name of a
+  .priv file in $HOME/.splinter/keys).
+
+`-U`, `--url` URL
+: Specifies the URL for the `splinterd` REST API. The URL is required unless
+  `$SPLINTER_REST_API_URL` is set.
+
+`-D`, `--display` DISPLAY-NAME
+: Specifies the display name for the created role.
+
+`--id-user` USER-ID
+: Specifies the user identity to update. Mutually exclusive to `--id-key`
+
+`--id-key` PUBLIC-KEY
+: Specifies the public key identity to update. Mutually exclusive to `--id-user`
+
+`--add-role` ROLE-ID 
+: Specifies a role to be add to the authorized identity. Specify multiple times
+  for more roles.
+
+`--rm-role` ROLE-ID
+: Specifies a role to be removed from the authorized identity. Specify multiple
+  times for more roles.
+
+
+EXAMPLES
+========
+This example updates an authorized identity with two assigned roles to include a
+third role.
```

Comment:
This doesn't describe the role removal in the command

Reply:
fixed